### PR TITLE
Fixes some major lag

### PR DESCRIPTION
**Pull request by ThatLing**
I think will get us our 99% LAGFREE back.

### CODE REVIEW

**Comment by GuppyLaxx** [comment]
chesse20 in disguise

**Comment by ghost** [comment]
When he was destroyed his essence infected thousands.

File: yogstation/code/modules/clothing/shoes/colour.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/obj/item/clothing/shoes/sneakers/white/pointed_at(var/mob/user)
```
Comment (Time-Green) [inline yogstation/code/modules/clothing/shoes/colour.dm:1]: You don’t need the var/ part, and pointed_at(mob/user) looks cooler

Comment (ThatLing) [reply yogstation/code/modules/clothing/shoes/colour.dm:1]: Don't question the chesse20 code, it's perfect like this

**Pull request merged**